elements/rh-table/docs/40-accessibility.md

```diff
@@ -75,14 +133,14 @@ Each cell includes enough spacing for selecting interactive elements.
 ## Additional guidelines
 
 - No column title cells should be blank
-- Column titles must use `<th>` elements with `scope="col"` attributes
 - Each cell should only have one piece of data
 - Do not place multiple inactive elements in a single cell
 
```

Comment:
Avoid merged cells.  When possible consider breaking a complex table into a series of simpler tables.